src/components/profile-header/asProfileHeader-styles.ts

```diff
@@ -0,0 +1,45 @@
+import { ImageStyle, StyleSheet, TextStyle, ViewStyle } from 'react-native'
+
+import { COLORS, Typography, SPACING } from '../../theme'
+
+interface Styles {
+  container: ViewStyle
+  bgImage: ImageStyle
+  header: ViewStyle
+  image: ImageStyle
+  title: TextStyle
+  icon: ViewStyle
+}
+
+const styles: Styles = StyleSheet.create({
+  container: {
+    flex: SPACING.space_1,
+  },
+  bgImage: {
```

Comment:
resize mode contain should be in the image props and use resizeMode={ResizeMode.Contain} (phele se enum bana rkha hai)